data-prepper-plugins/service-map-stateful/src/test/java/com/amazon/dataprepper/plugins/prepper/ServiceMapTestUtils.java

```diff
@@ -64,47 +53,42 @@
     }
 
     /**
-     * Creates a ResourceSpans object with the given parameters, with a single span
+     * Creates a {@link com.amazon.dataprepper.model.trace.Span} object with the given parameters
      * @param serviceName Resource name for the ResourceSpans object
      * @param spanName Span name for the single span in the ResourceSpans object
      * @param spanId Span id for the single span in the ResourceSpans object
      * @param parentId Parent id for the single span in the ResourceSpans object
      * @param spanKind Span kind for the single span in the ResourceSpans object
-     * @return ResourceSpans object with a single span constructed according to the parameters
-     * @throws UnsupportedEncodingException
+     * @return {@link com.amazon.dataprepper.model.trace.Span} object with a single span constructed according to the parameters
      */
-    public static ResourceSpans getResourceSpans(final String serviceName, final String spanName, final byte[]
-            spanId, final byte[] parentId, final byte[] traceId, final Span.SpanKind spanKind) throws UnsupportedEncodingException {
-        final ByteString parentSpanId = parentId != null ? ByteString.copyFrom(parentId) : ByteString.EMPTY;
-        return ResourceSpans.newBuilder()
-                .setResource(
-                        Resource.newBuilder()
-                                .addAttributes(KeyValue.newBuilder()
-                                        .setKey(OTelHelper.SERVICE_NAME_KEY)
-                                        .setValue(AnyValue.newBuilder().setStringValue(serviceName).build()).build())
-                                .build()
-                )
-                .addInstrumentationLibrarySpans(
-                        0,
-                        InstrumentationLibrarySpans.newBuilder()
-                                .addSpans(
-                                        Span.newBuilder()
-                                                .setName(spanName)
-                                                .setKind(spanKind)
-                                                .setSpanId(ByteString.copyFrom(spanId))
-                                                .setParentSpanId(parentSpanId)
-                                                .setTraceId(ByteString.copyFrom(traceId))
-                                                .build()
-                                )
-                                .build()
-                )
-                .build();
-    }
-
-    public static ExportTraceServiceRequest getExportTraceServiceRequest(ResourceSpans...spans){
-        return ExportTraceServiceRequest.newBuilder()
-                .addAllResourceSpans(Arrays.asList(spans))
-                .build();
+    public static Span getSpan(final String serviceName, final String spanName, final String
```

Comment:
This is never used, what is it for?

Reply:
It has been in use for constructing the test spans.